### ISSUE TITLE
update CRAN

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [ss3sim/ss3sim#376]
{bblme} is being archived on 2022-05-22, which the {ss3sim} CRAN version depends on. Thus, we will need to push the latest version of {ss3sim} to CRAN before then so we are also not archived. Consequently, @iantaylor-NOAA I will need {r4ss} to also be pushed to CRAN.

**Comment by iantaylor-NOAA** [comment]
I can work on the {r4ss} CRAN submission this week.
Just to clarify, does the current {ss3sim} no longer depend on {bblme} and that's why pushing to CRAN will avoid the archiving?

**Comment by kellijohnson-NOAA** [comment]
Exactly. Also, it appears that Ben Bolker is attempting to update {bblme}, and thus, archiving will not happen. Regardless, updating {ss3sim} and {r4ss} should occur soon.
